packages/app/src/portals/modal/index.tsx

```diff
@@ -90,17 +90,19 @@ type UseModalReturn = {
     onRequestClose: Props['onRequestClose'];
   };
 };
-export const useModal = (): UseModalReturn => {
+export const useModal = (onClose?: VoidFunction): UseModalReturn => {
```

Comment:
Considering the possibility of additional arguments in the future, I think it would be safer to structure it as an object.


```suggestion
export const useModal = ({ onClose }: { onClose?: VoidFunction} ): UseModalReturn => {
```